src/web/www/style/dynamics.css

```diff
@@ -152,6 +152,13 @@
   stroke-width: 1.2px;
 }
 
+.DHCode svg.warning {
+  fill: var(--warning-hole-fill);
+  stroke-dasharray: 1, 1;
+  stroke: var(--warning-hole-stroke);
+  stroke-width: 1.2px;
+}
+
```

Comment:
is this actually used? i thought DHCode stuff was dead code now. maybe check this